src/corona_hakab_model/analyzers/simulation_analysis.py

```diff
@@ -0,0 +1,31 @@
+import os
+from glob import glob
+from project_structure import OUTPUT_FOLDER
+import pandas as pd
+from datetime import datetime
+from itertools import chain
+
+
+def create_comparison_files(files: list = None):
+    """
+
+    :param files: Files to compare. If not given, takes all the files in the output folder.
+    :return: N/A. For each parameter creates a file combining the information from each file.
+    """
+    if files is None:
+        files = glob(os.path.join(OUTPUT_FOLDER, "*.csv"))
+    dfs_dict = dict(map(lambda f: (os.path.basename(f),
+                                   pd.read_csv(f).drop(["Unnamed: 0"], axis=1, errors='ignore')),
+                        files))
```

Comment:
use dictionary comprehension

Reply:
these files aren't mine, they are from develop (had these open already before i switched to the new matrix branch and pulled from dev-4.0).